### ISSUE TITLE
heartbeat/findif.sh: ifcheck must not suppress stdout

### ISSUE THREAD

**Issue by turchanov** [ClusterLabs/resource-agents#883]
ifcheck must not suppress stdout because a subsequent invocation in findif.sh:141
reads stdout to *errmsg* to print it out in case of an error:

```
  if [ -n "$nic" ] ; then
    errmsg=`ifcheck $nic`
    if [ $? -ne 0 ] ; then
      ocf_log err "Invalid interface name [$nic]: $errmsg"
      return $OCF_ERR_CONFIGURED
    fi
  fi
```

Proposed fix:
```
ifcheck() {
  local ifname="$1"
-  $IP2UTIL link show dev $ifname >/dev/null 2>&1
+  $IP2UTIL link show dev $ifname 2>&1
}
prefixcheck() {
```

**Comment by dmuhamedagic** [comment]
Good catch! Care to make a pull request? Thanks!

**Comment by turchanov** [comment]
Why not? Just did that. Pull #886

**Comment by krig** [comment]
Resolved by #866
